### ISSUE TITLE
Can't label button by Tooltip (Callout a11y issue)

### ISSUE THREAD

**Issue by Anschur** [OfficeDev/office-ui-fabric-react#10677]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out)

> Note that our screen reader support is limited to Edge + Narrator. Issues that do not reproduce in Edge + Narrator, and aren't caused by obvious invalid aria values, should be filed with the respective screen reading software, not the Fabric repo.

### Describe the issue:

I have a button that I have placed a tooltip on. In order for the tooltip to be read out when focused by Narrator I have used aria-labelledby and refered to the Tooltip. However the Tooltip text is not read when focusing the button.

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/svartn/pen/zYOVVqK

#### Actual behavior:

Narrator read "Question mark" - the text on the button.

#### Expected behavior:

Narrator read "This is the question"

**Comment by khmakoto** [comment]
Hi @Anschur, thanks for filing this! This is actually not a bug. What's happening here is that the `Tooltip` has a timer to appear specified by the `delay` property, which defaults to `300ms`. This means that when Narrator focuses on the `Button` the `Tooltip` has still not rendered, and therefore there is nothing the `aria-labelledby` is pointing to. To avoid this you can specify `delay={TooltipDelay.zero}` in the `TooltipHost` props so that the `Tooltip` renders immediately and Narrator gets the correct info on focus. I've forked your codepend [here](https://codepen.io/khmakoto/pen/zYOgqKQ) to show how this fixes the issue.

Please let me know if there is anything else I can do to help!

**Comment by Anschur** [comment]
Hi @khmakoto! Thanks for clarifying this :) 
I see many of the examples are using aria-describedby, but I guess you would have the same issue there? Maybe an idea to remove aria-descibedby from the examples if they are not working, and make e specific example on how to apply a working aria-describedby?

**Comment by khmakoto** [comment]
Hi @Anschur, yeah, the same issue happens there. Now that I think about it, this may actually be a bug because you expect the `Tooltip` to label things correctly independent of the `delay` you set. I'll have a PR to fix this and check that it works on the examples.

**Comment by Anschur** [comment]
Great! Thanks @khmakoto :)

**Comment by khmakoto** [comment]
@Anschur you're welcome!
